lib/datadog/statsd/serialization/event_serializer.rb

```diff
@@ -48,7 +48,11 @@ def format(title, text, options = EMPTY_OPTIONS)
             end
 
             if event.bytesize > MAX_EVENT_SIZE
-              raise "Event #{title} payload is too big (more that 8KB), event discarded"
+              if options[:truncate_if_too_long]
+                event.slice!(MAX_EVENT_SIZE..event.length)
```

Comment:
```suggestion
                event.slice!(MAX_EVENT_SIZE.event.length)
```

Maybe? Not sure if the 2 `.` were intentional.

Reply:
It is range expression